prow/jobs/api-gateway/api-gateway-build.yaml

```diff
@@ -84,6 +84,7 @@ postsubmits: # runs on main
       branches:
         - ^main$
         - ^release-\d+\.\d+$
+        - ^istio-jwt-handler$
```

Comment:
Would it be a good idea to use something like `feature-*`, so we could use feature branches in the future?

Reply:
this is really good point, totally makes sense. thx